include/seqan3/range/view/translation.hpp

```diff
@@ -45,292 +45,657 @@
 #include <seqan3/alphabet/nucleotide/dna5.hpp>
 #include <seqan3/alphabet/aminoacid/aa27.hpp>
 #include <seqan3/alphabet/aminoacid/translation.hpp>
-#include <seqan3/alphabet/aminoacid/translation_details.hpp>
+#include <seqan3/core/concept/core.hpp>
+#include <seqan3/core/add_enum_bitwise_operators.hpp>
 #include <seqan3/range/concept.hpp>
-#include <seqan3/range/view/complement.hpp>
+#include <seqan3/range/container/concept.hpp>
+#include <seqan3/range/view/detail.hpp>
+#include <seqan3/core/metafunction/range.hpp>
+#include <seqan3/range/detail/random_access_iterator.hpp>
 
-#include <range/v3/view/chunk.hpp>
-#include <range/v3/view/concat.hpp>
-#include <range/v3/view/drop.hpp>
-#include <range/v3/view/reverse.hpp>
-#include <range/v3/view/single.hpp>
-#include <range/v3/view/take_while.hpp>
-#include <range/v3/view/transform.hpp>
+namespace seqan3::detail
+{
+//!\brief Pop count function (calls gcc builtin function).
+template<integral_concept integral_type>
+auto pop_count(integral_type const n)
+{
+    return __builtin_popcount(n);
+}
+
+}
```

Comment:
Since this is only used in one place, you can just call the __builtin there directly. If we ever add a platform that doesn't support this the code will have to be refactored into an extra function somewhere else anyway!